changelog.txt

```diff
@@ -1,5 +1,8 @@
 *** WooCommerce Shipping & Tax Changelog ***
 
+= 2.7.1 - 2024-xx-xx =
+* Add - User interface components to perform the migration to the new WooCommerce Shipping plugin.
```

Comment:
I read this as the functionality already existed since it's a UI comment - can be ignored 😄 

```suggestion
* Add - A new migration experience from this plugin to the newly released WooCommerce Shipping plugin.
```

Reply:
Looks good to me, I am committing your suggestion.